Week1/MAKEME.md

```diff
@@ -1,183 +1,46 @@
-
-## Before you start with the homework:
-
-1. Go through the review of [week 1](https://github.com/HackYourFuture/JavaScript/blob/master/Week1/REVIEW.md)
-2. Watch: [What is programming](https://www.khanacademy.org/computing/computer-programming/programming/intro-to-programming/v/programming-intro) Just watch the 2 min video, you do not have to do the entire JavaScript course (It could be useful later on though). 
-3. Please watch the following parts of the course, [Programming Foundations Fundamentals](https://www.lynda.com/Programming-Foundations-tutorials/Welcome/83603/90426-4.html) on Lynda.com (if you don't have access to Lynda yet ask Gijs):
-    0. Introduction
-    1. Programming Basics
-    2. Core Programming Syntax 
-    3. Variables and Data Types
-
-## Step 1: Command Line
-
-_Deadline Tuesday_
-
->We covered a bit of command line usage in the first class and got a program running which is great. If you need a refresher for the command line please have a look here: https://github.com/HackYourFuture/CommandLine
-
-1. Create a `.js` file that prints `Hello` when you run it from the command line. (Hint: `node` is the program that can run your JavaScript files.)
-
 ```
-Write commands to do following:
-1. create a directory. Enter a directory. Create an empty file named blank. 
-    2. Then write the content `"Hello"` five times to the file greetings.txt. 
-   Then copy the file greetings.txt and paste its contents into 1.txt, 2.txt, 3.txt, 4.txt and 5.txt.
-    3. Then write the text "cat" to pets.txt
-   Then append the text "dog" to pets.txt
-   Then append the text "hamster" to pets.txt
-    4. Then write the text "cat" to commands.txt
-   Then append the text "ls" to commands.txt
-   Then append the text "pwd" to commands.txt
-    5. Then find unique strings from these two files pets.txt and commands.txt
-   and store the unique strings in lovelyCommands.txt
+Topics discussed in class this week:
+• Git
 ```
 
+>[Here](/Week1/README.md) you find the readings you have to complete before the fifth lecture.
 
-## Step 2: Feedback
-
-_Deadline Wednesday_
-
-Provide feedback on the HTML-CSS assignments (week 3) of one of your fellow students. You will be assigned to one of the assignments by the class lead of this week.
+## Step 1: Share a useful resource
 
-## Step 3: JavaScript
+_Deadline Monday_
 
-_Deadline Thursday_
+All share a video or a resource (this can be a drawing, article or a pod cast) that was helpful for you the last few weeks with learning JavaScript. Please share this in the channel of your class in Slack. Also write as small note about what the resource i about and why you think it's so helpful (you can share more than one if you like).
 
-> For all the following exercises create a new .js file. Try to find a proper name for each file or make a small comment about what it does inside for future reference. All these 
+## Step 2: Feedback
```

Comment:
Is there any home work from the preview from Week 0?

Reply:
yep, this refers to the homework of the third week of html/css/cli